src/transliterate/contrib/languages/hy/data/__init__.py

```
@@ -1,4 +1,16 @@
+    mapping,
+    pre_processor_mapping,
+    reversed_specific_mapping,
+    reversed_specific_pre_processor_mapping,
+)
+
+__all__ = (
+    'reversed_specific_pre_processor_mapping',
+    'mapping',
+    'reversed_specific_mapping',
+    'pre_processor_mapping',
+)
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)